README.md

```diff
@@ -19,7 +19,7 @@ Installation of Gophish is dead-simple - just download and extract the zip conta
 To build Gophish from source, simply run ```go get github.com/gophish/gophish``` and ```cd``` into the project source directory. Then, run ```go build```. After this, you should have a binary called ```gophish``` in the current directory.
 
 ### Docker
-You can also use Gophish via an unofficial Docker container [here](https://hub.docker.com/r/matteoggl/gophish/).
```

Comment:
Small nit: Could you update this to say “official” Docker container?

Reply:
Sure thing, committed!